lib/cassandra/cql.lua

```diff
@@ -1377,7 +1380,16 @@ do
     elseif op_code == OP_CODES.ERROR then
       local code = body:read_int()
       local message = body:read_string()
-      return nil, '['..ERROR_TRANSLATIONS[code]..'] '..message, code
+      local error_translation = ERROR_TRANSLATIONS[code]
+
+      -- If the translation is not found, return a formatted string
+      -- with the error code for convenience.
+      if error_translation == nil then
+        error_translation = string.format(
```

Comment:
We should be using the localized version of `string.format` (in the `fmt` local) instead here.